pkg/cloud/openstack/clients/machineservice.go

```diff
@@ -269,18 +273,64 @@ func (is *InstanceService) InstanceCreate(name string, config *openstackconfigv1
 		}
 	}
 	userData := base64.StdEncoding.EncodeToString([]byte(cmd))
-	createOpts = servers.CreateOpts{
+	clusterTag := ""
+	for _, tag := range config.Tags {
+		if strings.Contains(tag.Name, "ClusterID") {
```

Comment:
Can you elaborate on why you're adding the tag only for the `ClusterID`? Shouldn't we also support adding other tags?

Reply:
This was added so that resources created by CAPO can be deleted by the installer (following installer logic)

Reply:
We can simply add all, and that should work too